packages/core/helper-plugin/lib/src/components/ReactSelect/ReactSelect.js

```diff
@@ -9,14 +9,16 @@ import IndicatorSeparator from './components/IndicatorSeparator';
 
 import getSelectStyles from './utils/getSelectStyles';
 
-const ReactSelect = ({ components, styles, ...props }) => {
+const ReactSelect = ({ components, styles, error, ...props }) => {
   const theme = useTheme();
-  const customStyles = getSelectStyles(theme);
+  const customStyles = getSelectStyles(theme, error);
 
   return (
     <Select
       {...props}
       components={{ ClearIndicator, DropdownIndicator, IndicatorSeparator, ...components }}
+      aria-errormessage={error && 'folder-parent-error'}
```

Comment:
The error message doesn't make sense here: `ReactSelect` is meant to be a very generic component, which doesn't have anything todo with folders ...